html/Corporate/style.css

```diff
@@ -0,0 +1,675 @@
+            clip-path: polygon(0 0, 10% 0, 70% 100%, 0 100%);
+  }
+  .responsive-design__right-side {
+    -ms-flex-preferred-size: 55%;
+        flex-basis: 55%;
+    padding: 150px 10% 150px 0;
+    text-align: left;
+  }
+}
+
+@media screen and (min-width: 1600px) {
+  .responsive-design__laptop {
+    width: 600px;
+  }
+  .responsive-design__right-side {
+    padding: 200px 10% 200px 0;
+  }
+}
+
+.single-person__img {
+  width: 100px;
+  height: 100px;
+  margin: 0 auto;
+  background: url(./assets/ceo/entrepreneurship-3498259_640.jpg) no-repeat;
+  background-size: cover;
+  border-radius: 60px;
+}
+
+.single-person__name {
+  font-size: 1.2rem;
+  font-weight: normal;
+  margin: 10px 0 5px 0;
+  color: #ffffff;
+}
+
+.single-person__position {
+  color: #f04c4c;
+  font-size: 0.6rem;
+}
+
+.single-person__description {
+  font-size: 0.8rem;
+  line-height: 1.6;
+  opacity: 0.8;
+  margin-top: 20px;
+  color: #ffffff;
+  font-style: italic;
+}
+
+.founders-company__starbucks {
+  margin: 20px;
+  width: 130px;
+}
+
+.founders {
+  padding-left: 10%;
+  padding-right: 10%;
+  padding-top: 100px;
+  padding-bottom: 100px;
+  background: url(./assets/triangle-bg/group-4@3x.png) no-repeat;
+  background-size: cover;
+  text-align: center;
+}
+
+.founders__single-person {
+  margin-bottom: 70px;
+}
+
+@media screen and (min-width: 1024px) {
+  .founders-company__starbucks {
+    margin: 40px;
+  }
+  .founders__people {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: (1fr)[3];
+        grid-template-columns: repeat(3, 1fr);
+    grid-column-gap: 100px;
+  }
+}
+
+.awsome-feature {
+  padding-left: 10%;
+  padding-right: 10%;
+  padding-top: 100px;
+  padding-bottom: 100px;
+  background: #fcfcfb;
+  text-align: center;
+}
+
+.awsome-feature__type {
+  color: #f04c4c;
+  font-size: 0.6rem;
+  text-transform: uppercase;
+  letter-spacing: 3.2px;
+}
+
+.awsome-feature__title {
+  font-size: 1.7rem;
+  letter-spacing: 1.6px;
+  font-weight: normal;
+  margin: 10px 0 20px 0;
+}
+
+.awsome-feature__description {
+  font-size: 0.8rem;
+  line-height: 1.6;
+  opacity: 0.8;
+}
+
+.awsome-feature__screen {
+  display: none;
+  margin: 0 auto;
+  height: 400px;
+}
+
+@media screen and (min-width: 1024px) {
+  .awsome-feature {
+    padding-top: 100px;
+    padding-bottom: 0;
+  }
+  .awsome-feature__description {
+    width: 70%;
+    margin: 0 auto;
+  }
+  .awsome-feature__screen {
+    display: block;
+    margin-top: 100px;
+  }
+}
+
+.contact-form__input, .contact-form__submit {
+  width: 60%;
+  padding: 10px 10px;
+}
+
+.contact-form__input {
+  margin-bottom: 20px;
+  background: #323969;
+  border: 1px solid #ffffff;
+  border-radius: 3px;
+}
+
+.contact-form__input::-webkit-input-placeholder {
+  color: #ffffff;
+}
+
+.contact-form__input:-ms-input-placeholder {
+  color: #ffffff;
+}
+
+.contact-form__input::-ms-input-placeholder {
+  color: #ffffff;
+}
+
+.contact-form__input::placeholder {
```

Comment:
if this selectors should apply the same result write them in comma separated list like 
`.contact-form__input::-webkit-input-placeholder, 
.contact-form__input:-ms-input-placeholder
`